CMakeLists.txt

```diff
@@ -25,8 +25,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #
+project("TILEDB UNITTEST ARRAYS")
```

Comment:
Let's use `TileDBUnitTestArrays` to match the style of our other cmake project names.